proposals/nnnn-configurable-time-before-shutdown.md

```diff
@@ -0,0 +1,78 @@
+# Configurable time before shutdown
+
+* Proposal: [SDL-NNNN](nnnn-configurable-time-before-shutdown.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core]
+
+## Introduction
+
+To prevent missing logs in SDL shutdown add Additional ini file options:
+ - Write all logs to file system before shutdown 
+ - Maximum time of sdl shutting down
+ 
+## Motivation
+
+In debug mode SDL produce a lot of logs in is asynchronous mode. 
+In some use cases (like video streaming or big Put file) SDL produce a lot of logs.
+SDL collect this logs in queue and write to file system in separate thread.
+Writing to file system may requir big amount of time.
```

Comment:
@LuxoftAKutsan 
Writing to **the** file system may requir**e** big amount of time.